docs/tutorial/tutorial_basics.rst

```diff
@@ -511,3 +511,10 @@ if a particular :py:class:`Instrument` uses :py:data:`units` for the
 
 .. note:: While :py:class:`Meta` access is case-insensitive, :py:attr:`data`
 	  access is case-sensitive.
+
+Global metadata, which is consistent across all data variables for an instrument
```

Comment:
I think it would be helpful if you had an example of using the global metaheader to get something.